maven/pom.xml

```diff
@@ -5,36 +5,30 @@
     <groupId>org.cqfn.diktat</groupId>
     <artifactId>diktat-maven</artifactId>
     <packaging>pom</packaging>
-	<version>1.0.0-SNAPSHOT</version>
+    <version>1.0.0-SNAPSHOT</version>
 
 
     <distributionManagement>
-        <repository>
-            <id>artipie</id>
-            <url>https://central.artipie.com/akuleshov7/diktat</url>
-        </repository>
         <snapshotRepository>
-            <id>artipie</id>
-            <url>https://central.artipie.com/akuleshov7/diktat</url>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
         </snapshotRepository>
     </distributionManagement>
-    <repositories>
-        <!-- this is needed to use ktlint 0.37.1-fork -->
-        <repository>
-            <id>artipie</id>
-            <url>https://central.artipie.com/akuleshov7/diktat</url>
-        </repository>
-    </repositories>
+
     <pluginRepositories>
-        <!-- this is needed to use forked ktlint and diktat inside antrun plugin -->
         <pluginRepository>
-            <id>artipie</id>
-            <url>https://central.artipie.com/akuleshov7/diktat</url>
+            <!-- for dokka and probably other kotlin-related artifacts -->
+            <id>jcenter</id>
```

Comment:
I'm not sure jcenter is needed to *run* diktat, it may be is only used during build. Did you check if it works without it? And if it is needed, it's most probably because of something other than dokka.

Reply:
Works without it